java/flight/flight-sql/src/test/protobuf/flightSQLExample.proto

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
nit: be consistent with file naming.  Uppercase Flight

Reply:
Additionally shouldn't Protobuf file names be in snake_case? See https://developers.google.com/protocol-buffers/docs/style#file_structure

Reply:
I guess Flight.proto itself is already an exception...we should be consistent with what we have, though, yes (so `FlightSQLExample.proto` or preferably `FlightSql.proto` and `FlightSqlExample.proto`)

Reply:
Yes I this is why I didn't comment on file naming